pypi_formula_mappings.json

```diff
@@ -81,7 +81,8 @@
     "extra_packages": ["robotframework-archivelibrary", "robotframework-selenium2library", "robotframework-sshlibrary"]
   },
   "salt": {
-    "extra_packages": ["M2Crypto", "pygit2"]
+    "extra_packages": ["pycryptodome", "pygit2"],
+    "exclude_packages": ["pyobjc"]
```

Comment:
It excludes only `pyobjc` resource, but a bunch of other `pyobjc-*` resources will be added, unfortunately.
I guess we could add support for wildcards for excluded package

Reply:
Would be nice, yes, but I suspect we'd need a lot of extra logic in `update-python-resources` to support wildcard tokens

Reply:
Damn, I was hoping this would work as sort of a metapackage. Yeah I think wildcards would be needed for anything that tries to use `pyobjc-*` since that rarely ever works on the latest macOS

Reply:
Next salt release is expected in March 31st, and all `pyobjc-*` have been removed: https://github.com/saltstack/salt/blob/v3003rc1/requirements/static/pkg/py3.9/darwin.txt

Maybe this can help to avoid this issue, although it sounds reasonable to add support to Homebrew for wildcard tokens.